server/graphql/schema/unit/resolvers/queries/searchIngestionSubjects.ts

```diff
@@ -14,6 +15,9 @@ export default async function searchIngestionSubjects(_: Parent, args: QuerySear
 
     const results: DBAPI.SubjectUnitIdentifier[] = [];
     const resultSet: Set<string> = new Set<string>();
+    console.log(`db: ${JSON.stringify(resultsDB)}`);
```

Comment:
While previously involved in the project, my suggestion was to not leave console logging in our committed code.  For server-side logging, use the logger.  For client-side logging, comment out console statements before committing.  Perhaps guidance has changed here, but I would still strongly suggest using server logging instead of console.log.